controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -218,14 +218,14 @@ def _mock_cli_object(cli_object):
         return Mock(as_single_element=cli_object)
 
     @staticmethod
-    def _get_cli_volume():
+    def _get_cli_volume(se_copy='yes', deduplicated_copy='no'):
```

Comment:
It's better to have booleans as default parameters than to expose the internal data structure.

Reply:
done